src/main/resources/application.yml

```diff
@@ -76,3 +76,10 @@ springdoc:
 
 server:
   forward-headers-strategy: framework
+
+mongock:
+  enabled: true
+  change-logs-scan-package:
+    - nl.dtls.fairdatapoint.database.mongo.migration.production
```

Comment:
This is duplicated in `MongoConfig.java`. Could we get away with just one definition?

Reply:
Good catch... added also legacy migration to `MongoConfig.java` and nothing about Mongock is in `application.yml`